### ISSUE TITLE
Problems with Spyder on Mac

### ISSUE THREAD

**Issue by maxbrazhnyy** [spyder-ide/spyder#14069]
### What steps will reproduce the problem?

When I enter spyder is suggests that I update it and then a mac update window appears saying that the commant "git" requires the installation of the command line tools. After I press install, is says that it´s looking for the software and then that it wasn´t possible to intall it because it´s not available on the server. When I press "ok", a notification from the MacOS settings appears and when I go to the Software Update part, ist says that I have an update to make but it either disappears shortly after or, if I try to install it, it cancels the installation by itself with the same message as before. All this process happens every single time I open spyder or change the tab on the left side of the program window.

## Versions

* Spyder version: 4.1.4 
* Python version: 3.8.1
* Qt version: 5.9.7
* PyQt5 version: 5.9.2
* Operating System: Darwin 19.6.0

### Dependencies

```

# Mandatory:
applaunchservices >=0.1.7      :  0.2.1 (OK)
atomicwrites >=1.2.0           :  1.4.0 (OK)
chardet >=2.0.0                :  3.0.4 (OK)
cloudpickle >=0.5.0            :  1.5.0 (OK)
diff_match_patch >=20181111    :  20200713 (OK)
intervaltree                   :  None (OK)
IPython >=4.0                  :  7.16.1 (OK)
jedi =0.17.1                   :  0.17.1 (OK)
nbconvert >=4.0                :  5.6.1 (OK)
numpydoc >=0.6.0               :  1.1.0 (OK)
parso =0.7.0                   :  0.7.0 (OK)
pexpect >=4.4.0                :  4.8.0 (OK)
pickleshare >=0.4              :  0.7.5 (OK)
psutil >=5.3                   :  5.7.0 (OK)
pygments >=2.0                 :  2.6.1 (OK)
pylint >=1.0                   :  2.5.3 (OK)
pyls >=0.34.0;<1.0.0           :  0.34.1 (OK)
qdarkstyle >=2.8               :  2.8.1 (OK)
qtawesome >=0.5.7              :  0.7.2 (OK)
qtconsole >=4.6.0              :  4.7.5 (OK)
qtpy >=1.5.0                   :  1.9.0 (OK)
rtree >=0.8.3                  :  0.9.4 (OK)
sphinx >=0.6.6                 :  3.1.2 (OK)
spyder_kernels >=1.9.2;<1.10.0 :  1.9.2 (OK)
watchdog                       :  None (OK)
zmq >=17                       :  19.0.1 (OK)

# Optional:
cython >=0.21                  :  0.29.21 (OK)
matplotlib >=2.0.0             :  3.2.2 (OK)
numpy >=1.7                    :  1.18.5 (OK)
pandas >=0.13.1                :  1.0.5 (OK)
scipy >=0.17.0                 :  1.5.0 (OK)
sympy >=0.7.3                  :  1.6.1 (OK)
```

**Comment by steff456** [comment]
Hi @maxbrazhnyy, 

How did you installed Spyder? Please try to install spyder in a new environment and tell me if the error persist,

```
conda create -n new-env python=3.8
conda activate new-env
conda install spyder
```

**Comment by ccordoba12** [comment]
Hey @steff456, thanks for giving support to @maxbrazhnyy. However, this is a duplicate of issue #12045 and @juanis2112 is working to fix it.

The fix will be available in our next version (4.2.0) to be released in two weeks or so. In the meantime there's nothing you can do about it, sorry.
